logtail.js

```diff
@@ -53,21 +57,23 @@ function get_log() {
                 if (!c_r)
                     throw "Server did not respond with a Content-Range";
 
-                size = parseInt(c_r.split("/")[1]);
-                if (isNaN(size))
+                log_file_size = parseInt(c_r.split("/")[1]);
+                content_size = xhr.getResponseHeader("Content-Length");
+                
+                if (isNaN(log_file_size))
                     throw "Invalid Content-Range size";
             } else if (xhr.status === 200) {
-                if (log_size > 1)
+                if (!first_load)
                     throw "Expected 206 Partial Content";
 
```

Comment:
Perhaps this should be changed as well. If the file is truncated, and `first_load` is `false`, this will throw and stop the loop. I think it's better to reset and continue.
```
log_file_size = 0;
log_data = "";
show_log();
```